### ISSUE TITLE
The "Return to the Fold" quest can't be completed due to its first objective not working

### ISSUE THREAD

**Issue by vampirika** [AzerothWarsLR/WarcraftLegacies#3172]
![image](https://github.com/user-attachments/assets/4ffd0c24-4bec-4f77-96af-b55beb23d5d7)

**Comment by YakaryBovine** [comment]
This is a duplicate of #3154, but I'll close that one because this is clearer.

To replicate:
1. As Sentinels, complete the quest "Shores of Feathermoon"
2. Observe the "Object reference not set to an instance of an object" error that pops up

The issue is exactly what it says. This is the code for the quest's `OnComplete`: 
```cs
    /// <inheritdoc />
    protected override void OnComplete(Faction completingFaction)
    {
      completingFaction.Player?
        .RescueGroup(_rescueUnits);
    }
```
`_rescueUnits` is never populated, so this doesn't work. In fact there are no units in Feathermoon Stronghold, so the method should just be removed and the tooltip for the quest updated accordingly.

**Comment by YakaryBovine** [comment]
Closed by #3177
